firefox-ios/EcosiaTests/Account/Auth/NativeToWebSSOAuth0ProviderTests.swift

```diff
@@ -22,13 +22,6 @@ final class NativeToWebSSOAuth0ProviderTests: XCTestCase {
         )
     }
 
-    override func tearDown() {
-        provider = nil
-        mockSettings = nil
-        mockCredentialsManager = nil
-        super.tearDown()
-    }
```

Comment:
Why can't we clean up between tests here?

Reply:
No need as the `setUp()` is re-initializing the properties anyway.